lib/cli/ui/prompt.rb

```diff
@@ -48,26 +49,29 @@ class << self
         # Free form question when the answer can be empty
         #   CLI::UI::Prompt.ask('What is your opinion on this question?', allow_empty: true)
         #
-        # Question with answers
-        #   CLI::UI::Prompt.ask('What kind of project is this?', options: %w(rails go ruby python))
+        # Multiple choice question with defined handlers
+        #   CLI::UI::Prompt.ask('What kind of project is this?', options: %w(rails go ruby python)) do |handler|
```

Comment:
It looks like `options:` is supposed to go away, right?

Reply:
Yep! You're right haha, I missed that.